cypress/integration/chi-js/mobile-nav.spec.js

```diff
@@ -0,0 +1,175 @@
+const drawerDropDownItem   = '.chi-mobile-nav__list-item-list ul.chi-tabs li';
+
+describe('mobile-nav-menu', function() {
+  before(() => {
+    cy.visit('/tests/js/mobile-nav.html');
+  });
+  describe('mobile-navigation-interaction-trigger button is present', () => {
+    it('mobile-navigation-interaction-trigger button should be visible', () => {
+      cy.get('#mobile-navigation-interaction-trigger', { timeout: 30000 })
```

Comment:
Timeout is not necessary. Let's remove it please.

Reply:
Hi @janibolkvadze,
 removed the timeout